app/Controllers/apiController.php

```diff
@@ -41,7 +41,7 @@ public static function updatePassword(string $apiPasswordPlain) {
 	 */
 	public function updatePasswordAction(): void {
 		if (!FreshRSS_Auth::hasAccess()) {
-			Minz_Error::error(403);
+			Minz_Error::error(FreshRSS_HttpResponseCode::FORBIDDEN);
```

Comment:
Granted, my brain developed its own HTTP status code parser literal decades ago, but I find the former immeasurably clearer than the latter.

Reply:
That's not necessarily a comment on the principle but it's definitely a comment on the execution. :-)

Reply:
I just realized I'm not offering a concrete solution, but a much clearer pattern exists in the form of for example `HTTP_400_BAD_REQUEST`, `HTTP_422_UNPROCESSABLE_ENTITY`, etc.

See https://www.django-rest-framework.org/api-guide/status-codes/

Reply:
I have updated following your recommendations